module/cyphersystem.js

```diff
@@ -784,26 +784,26 @@ function allInOneRollDialog(actor, pool, skill, assets, effort1, effort2, additi
   function applyToMacro(pool, skill, assets, effort1, effort2, additionalCost, additionalSteps, stepModifier, title, damage, effort3, damagePerLOE) {
     let cost = parseInt(additionalCost);
     let effort = parseInt(effort1) + parseInt(effort2) + parseInt(effort3);
-    if (stepModifier == "hindered") additionalSteps = parseInt(additionalSteps) * -1;
+    if (stepModifier.toLowerCase() == "hindered") additionalSteps = parseInt(additionalSteps) * -1;
     let modifier = parseInt(skill) + parseInt(assets) + parseInt(effort1) + parseInt(additionalSteps);
-    let skillRating = "Practiced";
-    let steps = " steps";
-    let points = " points";
-    let rollEffort = " levels";
-    let otherEffort = " levels";
-    let damageEffortLevel = " levels";
+    let skillRating = game.i18n.localize("CYPHERSYSTEM.Practiced");
+    let steps = " " + game.i18n.localize("CYPHERSYSTEM.Steps");
+    let points = " " + game.i18n.localize("CYPHERSYSTEM.Points");
+    let rollEffort = " " + game.i18n.localize("CYPHERSYSTEM.Levels").toLowerCase();
+    let otherEffort = " " + game.i18n.localize("CYPHERSYSTEM.Levels").toLowerCase();
+    let damageEffortLevel = " " + game.i18n.localize("CYPHERSYSTEM.Levels").toLowerCase();
```

Comment:
Note to self: If we have the strings in the translation already, we don’t need to define them here separately.

Reply:
Nothing for me there but we should potentially change line 860 with a nested translation.